src/interop/op-supervisor/op_supervisor_launcher.star

```diff
@@ -73,13 +73,15 @@ def launch(
     )
 
     service = plan.add_service(interop_constants.SUPERVISOR_SERVICE_NAME, config)
+    service_url = util.make_service_http_url(service)
 
-    observability.register_op_service_metrics_job(
-        observability_helper,
-        service,
-    )
+    if observability_helper.enabled:
+        observability.register_op_service_metrics_job(
+            observability_helper,
+            service,
+        )
 
-    return "op_supervisor"
```

Comment:
I would either strip this or return `struct(service=service)` - we will be making these return values more consistent

Reply:
I merged the latest changes, it should be ready to review